packages/babel-parser/ast/spec.md

```diff
@@ -1364,6 +1365,17 @@ interface ExportSpecifier <: ModuleSpecifier {
 
 An exported variable binding, e.g., `{foo}` in `export {foo}` or `{bar as foo}` in `export {bar as foo}`. The `exported` field refers to the name exported in the module. The `local` field refers to the binding into the local module scope. If it is a basic named export, such as in `export {foo}`, both `exported` and `local` are equivalent `Identifier` nodes; in this case an `Identifier` node representing `foo`. If it is an aliased export, such as in `export {bar as foo}`, the `exported` field is an `Identifier` node representing `foo`, and the `local` field is an `Identifier` node representing `bar`.
 
+### ExportNamespaceSpecifier
+
+```js
+interface ExportNamespaceSpecifier <: ModuleSpecifier {
+  type: "ExportNamespaceSpecifier";
+  exported: Identifier;
+}
+```
+
+A namespace export specifier, e.g., `* as foo` in `export * as foo from "mod.js"`.
```

Comment:
Per @JLHwung request, please add this note:

If an \`ExportNamedDeclaration\` contains \`ExportNamespaceSpecifier\`, it must have only one \`ExportNamespaceSpecifier\`.